packages/peregrine/lib/talons/MagentoRoute/useMagentoRoute.js

```diff
@@ -1,61 +1,98 @@
 import { useEffect, useRef, useState } from 'react';
-import { useLocation } from 'react-router-dom';
+import { useHistory, useLocation } from 'react-router-dom';
 import { useApolloClient } from '@apollo/react-hooks';
 
 import addToCache from './addToCache';
 import getRouteComponent from './getRouteComponent';
 
+const CODE_PERMANENT_REDIRECT = 301;
+const CODE_TEMPORARY_REDIRECT = 302;
+const REDIRECT_CODES = [CODE_PERMANENT_REDIRECT, CODE_TEMPORARY_REDIRECT];
+
+const DEMO = new Map().set('/penelope-peasant-blouse.html', {
+    isRedirect: true,
+    relativeUrl: '/susanna-draped-tank.html'
+});
```

Comment:
```suggestion
```

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/2504/commits/5be73656c08286410d34942110d9546cf67f28b0.